config/prod/prow/config_knative.yaml

```diff
@@ -319,13 +319,16 @@ presubmits:
   knative/test-infra:
   - build-tests: true
   - unit-tests: true
+  - go-coverage: true
+  knative/hack:
+  - build-tests: true
```

Comment:
I was thinking not to use prow for hack repo, just GitHub actions.

Reply:
We are running some smoke integration tests with GKE there to make sure the basic e2e test flow is not broken, and also there are some bash unit tests, since we are still running Prow jobs for all Knative repos, I think Prow job is better than GitHub Actions for these cases?

Reply:
I do not think we should put those tests inside of the hack repo. Keep them in test-infra for a smoke test and test-infra can pull in hack

Reply:
Then we are not running presubmit tests for the scripts in the `hack` repo, which meant if the base e2e test flow is broken, we will only find it out after the scripts are updated in `test-infra` repo.